Yoast/wordpress-seo
# Remove meta keywords functionality

## Summary

This PR can be summarized in the following changelog entries:

* Removes all uses of meta keywords throughout the plugin.
* Removes the "noindex subpages" feature as Google has gotten much better at paginated series, and it's now actually detrimental to use.
* Because both features above are removed, removes the "Other" tab from the Titles & Metas settings screen.

## Relevant technical choices:

* Deprecated the class-frontend function for meta keywords.
* Added an upgrade function for 6.3 that removes the related settings from the `wpseo_titles` option.

## Test instructions

This PR can be tested by following these steps:

* Meta keywords should be gone _everywhere_.
* Noindex subpages feature is gone.
* Related options in the `wpseo_titles` option should be gone, _if_ the upgrade routine has run (for which you have to set your version to 6.3).

## Quality assurance


Fixes: #8753 
Fixes: #8709
